filterpage.html

```diff
@@ -0,0 +1,367 @@
+        width: 120px;
+      }
+
+      .open-filter__btn,
+      .apply-filters__btn {
+        background-color: #ea2525;
+        border-radius: 5px;
+        border: none;
+        font-size: 15px;
+        font-weight: 600;
+        color: #ffffff;
+        padding: 12px;
+        margin-top: 10px;
+      }
+
+      .selected-filters {
+        margin-top: 15px;
+        font-size: 20px;
+        font-weight: 600;
+        color: #242323;
+      }
+
+      .selection-status,
+      .body-selection-status {
+        margin-top: 10px;
+        font-size: 20px;
+        font-weight: 300;
+        color: #5d5a5a;
+      }
+
+      .cross-symbol-top {
+        height: 35px;
+        width: 35px;
+        border-radius: 50%;
+        background-color: #ffffff;
+        display: flex;
+        justify-content: center;
+        border: none;
+        align-items: center;
+        box-shadow: 0 2px 2px 0 rgba(138, 138, 138, 0.2),
+          0 6px 20px 0 rgba(170, 169, 169, 0.19);
+      }
+
+      .cross-symbol-img {
+        height: 20px;
+        width: 20px;
+      }
+
+      .filter-image-headline-wrappper {
+        display: flex;
+        padding-left: 10px;
+      }
+
+      .filter-headline {
+        font-size: 24px;
+        font-weight: 800;
+        color: #3c3b3b;
+        padding-left: 15px;
+        padding-top: 5px;
+      }
+
+      .make-car-list,
+      .body-wrapper {
+        border: 1px solid #e0dede;
+        border-radius: 5px;
+        margin: 10px;
+        padding: 15px;
+      }
+
+      .selection-wrapper {
+        position: fixed;
+        top: 0;
+        height: 100%;
+        background-color: white;
+        width: 100%;
+      }
+      .checkbox-wrapper {
+        padding: 5px;
+        list-style: none;
+        font-family: arial;
+      }
+
+      .make-heading,
+      .body-heading {
+        font-size: 20px;
+        font-weight: 500;
+        color: #302a2a;
+        font-family: Arial, Helvetica, sans-serif;
+      }
+
+      .apply-filters__btn {
+        width: calc(100% - 20px);
+        position: absolute;
+        right: 10px;
+        left: 10px;
+        bottom: 5px;
+      }
+
+      .make-and-arrow-wrapper,
+      .body-and-arrow-wrapper {
+        display: flex;
+        justify-content: space-between;
+      }
+
+      .body-list-wrapper--content,
+      .make-list-wrapper--content {
+        width: 50%;
+      }
+
+      .arrow,
+      .body-arrow {
+        height: 20px;
+      }
+
+      .hide {
+        display: none;
+      }
+
+      .make-list-wrapper,
+      .body-list-wrapper {
+        display: flex;
+        flex-wrap: wrap;
+        margin-top: 10px;
+        /* -webkit-transition: all 0.5s ease;
+        -o-transition: all 0.5s ease;
+        transition: all 0.5s ease; */
+      }
+
+      .collapse {
+        height: 0;
+        overflow: hidden;
+        -webkit-transition: all 1s ease;
+        -o-transition: all 1s ease;
+        transition: all 1s ease;
```

Comment:
Again is this transition working as expected?

Reply:
yes now it's working I have added height